### PR TITLE
Qol patch

### PR DESCRIPTION
**Pull request by MeatyLearner**
-Made prop standing unequip weapons and cosmetics
-Fixed ghosts being able to bet slime at some casino games
-Fixed a fishing exploit where casting and reeling multiple times would cause the player to be fishing multiple times at once
-Changed weapon skill gain to require the shootee to also be higher levelled than the shooter's weapon skill

### CODE REVIEW

File: ewcfg.py [hunk 1]
```diff
@@ -8095,7 +8097,8 @@ def atf_hooves(ctn=None):
 		role = "Juvie's Row",
 		pvp = False,
 		property_class = property_class_b,
-		community_chest = chest_id_juviesrow
+		community_chest = chest_id_juviesrow,
+		pier_type = fish_slime_saltwater
```
Comment (magicksRampage) [inline ewcfg.py]: wouldn't it make more sense to put this info with the pier pois?

**Review by magicksRampage — CHANGES_REQUESTED** [review]

File: ewcfg.py (hunk 1 above)
Comment (MeatyLearner) [reply ewcfg.py]: pier pois aren't used since we removed subzones, so when a fisher !casts their assigned pier is the mother district

Comment (MeatyLearner) [reply ewcfg.py]: i could use `chname_to_poi` to get the pier poi, if you'd prefer it that way

Comment (magicksRampage) [reply ewcfg.py]: why though? what if we want to have multiple piers in the same location?

Comment (magicksRampage) [reply ewcfg.py]: > i could use `chname_to_poi` to get the pier poi, if you'd prefer it that way

yeah, I'd like that

File: ewfish.py [hunk 2]
```diff
@@ -239,12 +251,12 @@ def gen_fish(x, cmd, has_fishingrod):
 			if ewcfg.fish_map[fish].catch_time != None:
 				fish_pool.remove(fish)
 
-	if cmd.message.channel.name in ["slimes-end-pier", "assault-flats-beach-pier", "vagrants-corner-pier", "ferry"]:
+	if cmd.message.channel.name in ["slimes-end-pier", "assault-flats-beach-pier", "juvies-row-pier", "ferry"]:
```
Comment (magicksRampage) [inline ewfish.py:254]: why do we have the pier_type attribute if we don't use it for this?

**Pull request merged**